openedx/core/djangoapps/credit/tests/test_api.py

```diff
@@ -623,7 +623,7 @@ def test_satisfy_all_requirements(self):
         self.assertFalse(api.is_user_eligible_for_credit(user.username, self.course_key))
 
         # Satisfy the other requirement
-        with self.assertNumQueries(21):
+        with self.assertNumQueries(22):
```

Comment:
Indeed. This test has been balking at me for the last couple of runs. I *think* it's an issue with rebasing, but I'll run tests a few times to confirm.